Kitodo/src/main/java/org/kitodo/production/services/workflow/WorkflowControllerService.java

```diff
@@ -211,15 +211,13 @@ public void closeTaskByUser(Task task) throws DataException, IOException, DAOExc
                 if (!mih.checkIfImagesValid(task.getProcess().getTitle(), imageFolder)) {
                     throw new DataException("Error on image validation!");
                 }
+                this.webDav.uploadFromHome(task.getProcess());
```

Comment:
This move is wrong as the method is only called when the task should be verified on closing and the images could be modified.

I would recommend to 
- not move the call to this place
-  on original place wrap the call with a test to `task.isTypeImagesRead() || task.isTypeImagesWrite()`

Reply:
By the way, @Kathrin-Huber, could you maybe rename the method? Nothing is uploaded there. No idea how, but that name doesn't make any sense.

Reply:
Not only the method even the whole class WebDav did not make this what the name is suggesting. Renaming should be done after this pull request is merged.